bitnami/fluentd/README.md

```diff
@@ -234,39 +260,3 @@ $ helm install bitnami/fluentd \
     --set aggregator.extraEnv[1].name=ELASTICSEARCH_PORT \
     --set aggregator.extraEnv[1].value=your-port-here \
 ```
```

Comment:
IMHO it's interesting to maintain those CLI commands to show users how to use this configmap

Reply:
I would remove the command and mention the parameters the user needs to configure.